sdk/analytics/src/main/java/com/klaviyo/analytics/networking/KlaviyoApiClient.kt

```diff
@@ -257,7 +263,7 @@ internal object KlaviyoApiClient : ApiClient {
                         break
                     }
                     // Offline or at the end of the queue... either way break the loop
-                    null, KlaviyoApiRequest.Status.Unsent -> break
+                    else -> break
```

Comment:
Do you think it might be better to be exhaustive in case you add more cases?

Reply:
but that'd be like the safer and smarter thing to do